tests/basic.t

```diff
@@ -19,7 +19,7 @@
 HOST=$(hostname -I | awk '{print $1}')
 VOLNAME="hosting-volume"
 BLKNAME="block-volume"
-BRKDIR="/brick"
+BRKDIR="/data/brick"
```

Comment:
You should 'mkdir /data/' somewhere 
Else volume create will fail IMO.

Reply:
Yeah, will fix it.
Thanks.